src/hooks/useGallery.ts

```diff
@@ -23,7 +23,7 @@ export const useGallery = (userId: string) => {
       },
       async ({ data, error, onNextPage, hasNextPage }) => {
         if (error) return null;
-        hasNextPage ? setGetNextPage(onNextPage) : setGetNextPage(null);
+        hasNextPage ? setGetNextPage(() => onNextPage) : setGetNextPage(null);
```

Comment:
How it different from the original one

Reply:
Before that, it runs itself whenever we have `hasNextPage`. 
So that, I have to use as callback just only to save the function